bears/c_languages/GNUIndentBear.py

```diff
@@ -23,18 +23,169 @@ class GNUIndentBear:
     def create_arguments(filename, file, config_file,
                          max_line_length: int=80,
                          use_spaces: bool=True,
+                         blank_lines_after_declarations: bool=False,
```

Comment:
okay :)

2016-06-17 14:56 GMT+01:00 Abhay Raizada <notifications@github.com>:

> In bears/c_languages/GNUIndentBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/519#discussion_r67513685>
> :
>
> > @@ -23,18 +23,169 @@ class GNUIndentBear:
> >      def create_arguments(filename, file, config_file,
> >                           max_line_length: int=80,
> >                           use_spaces: bool=True,
> > +                         blank_lines_after_declarations: bool=False,
>
> maybe it should, i guess its okay for now, maybe we should open an issue
> to order all params in lint bears alphabetically.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/519/files/4e617267213ef6a8fbac778c54e69f17da5bf6d9#r67513685>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ9xokn6PYeQhkH1UojWWolQ4lgan1CBks5qMqd0gaJpZM4I02l0>
> .
>